satellite/metainfo/config.go

```diff
@@ -18,7 +18,7 @@ const (
 
 // Config is a configuration struct that is everything you need to start a metainfo
 type Config struct {
-	DatabaseURL          string      `help:"the database connection string to use" default:"bolt://$CONFDIR/pointerdb.db"`
+	DatabaseURL          string      `help:"the database connection string to use" default:"bolt://$CONFDIR/pointerdb.db" mandatory:"true"`
```

Comment:
Should `mandatory` invalidate the `default` value?

As far I understand if the `default` isn't an empty string, the `mandatory` check won't apply because the default value is used and it isn't empty.